packages/editor/src/components/provider/use-block-editor-settings.js

```diff
@@ -215,6 +214,10 @@ function useBlockEditorSettings( settings, postType, postId ) {
 		[ saveEntityRecord, userCanCreatePages ]
 	);
 
+	const syncGetLinkSuggestions = useCallback( async ( ...args ) => {
+		return await resolveSelect( coreStore ).getLinkSuggestions( ...args );
```

Comment:
If we make this private how do we `unlock` a `resolveSelect`?

Reply:
Why not use a normal select call in a `useSelect`?

Reply:
If we did this then we'd effectively pass something like `getEntityRecords` as a setting. We can't pass the result of a selector call because we need different args per invocation.

I think I tried doing this and found it didn't work. I could be wrong - would you expect it to?